scripts/screeps_tencent_batch_rl_runner.py

```
@@ -0,0 +1,1017 @@
+    def safe_scale_down(self) -> None:
+        try:
+            self.scale_down()
+        except Exception as error:
+            self.result["scaleDownError"] = type(error).__name__ + ": " + str(error)
+            if self.final_status == "completed":
+                self.final_status = "completed_scale_down_failed"
+
+    def ensure_map_present(self) -> None:
+        target = REPO_ROOT / "maps" / "map-0b6758af.json"
+        if target.is_file():
+            self.record_step("map_preflight", time.time(), True, detail={"path": str(target)})
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)